frontend/src/components/templates/Login/index.js

```diff
@@ -74,16 +75,8 @@ export const loginUrl = `${BACK_HOST}/users/auth/api_gouv${hashToQueryParams({
 
 const LoginButtons = ({ isOnNewEnrollmentPage }) => (
```

Comment:
I think that you can remove the LoginButtons component and put the form in the MonComptePro component